SwanContents/swancontents/swanclassic/handlers/share.py

```diff
@@ -1,10 +1,16 @@
-from tornado import web
-from notebook.base.handlers import IPythonHandler, path_regex
-from notebook.utils import url_path_join, url_escape
-from ..filemanager.fileio import swan_sharing_folder
+from tornado import web, gen
 
+from jupyter_server.base.handlers import JupyterHandler
+from jupyter_server.extension.handler import (
+    ExtensionHandlerMixin,
+    ExtensionHandlerJinjaMixin
+)
+from jupyter_server.base.handlers import path_regex
+from jupyter_server.utils import url_path_join, url_escape, ensure_async
+from swancontents.filemanager.fileio import swan_sharing_folder
```

Comment:
There's no problem with an _explicit_ relative import, as you had initially, and actually you should use it if you can. What is forbidden in Python3 is an _implicit_ relative import (see [this](https://docs.python.org/release/3.0.1/whatsnew/3.0.html#removed-syntax)).